### PR TITLE
 Space-Travelers-Hub:Fetch missions

### PR DESCRIPTION
**Pull request by GedeonTS**
In this pull reauest I Fetched data from the Missions endpoint  (https://api.spacexdata.com/v3/missions) when a user navigates to the Missions section.

Once the data are fetched, dispatch an action to store the selected data in Redux store:

- mission_id
- mission_name
- description

### CODE REVIEW

File: src/redux/missions/missions.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
+import axios from 'axios';
+
+const FETCH_MISSIONS = 'bookstore/src/redux/books/FETCH_MISSIONS';
```
Comment (Estherstarr) [inline src/redux/missions/missions.js:3]: I think naming the folder and files with names relating to the project will be preferred for clarity.
Eg 'spaceTravellersHub/src/redux/missions/FETCH_MISSIONS'

File: src/redux/missions/missions.js [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+const LAND_SURVEY = 'bookstore/src/redux/books/LAND_SURVEY';
+
+const URL = 'https://api.spacexdata.com/v3/missions';
+
+const init = {
+  name: 'hector', idea: 'remberence', field: 'air conditioning', status: 'research',
+};
+
+// const joinMission = <button></button>
```
Comment (Estherstarr) [inline src/redux/missions/missions.js:12]: Kindly remove comment

**Comment by Estherstarr** [comment]
Hi @GedeonTS , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
- Linters are set up properly ✔️.


## Required Changes ♻️
Check the comments under the review.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

File: src/redux/missions/missions.js (hunk 2 above)
Comment (GedeonTS) [reply src/redux/missions/missions.js:12]: good

**Comment by GedeonTS** [comment]
thanks for the feedback!

**Pull request merged**